<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeWeek</class>
 <widget class="QFrame" name="TimeWeek">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="oneKey">
       <property name="text">
        <string>一键将周二到周日设置为与周一相同数据</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>周一</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_1"/>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>周二</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_2"/>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>周三</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_3"/>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>周四</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_4"/>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>周五</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_5"/>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>周六</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_6"/>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>周日</string>
      </attribute>
      <layout class="QVBoxLayout" name="vBox_7"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
